src/components/ReleaseService/ReleasePlan/TriggerRelease/TriggerReleaseForm.tsx

```diff
@@ -21,11 +23,19 @@ export const TriggerReleaseForm: React.FC<Props> = ({
   dirty,
   errors,
   status,
-  applicationName,
 }) => {
   const breadcrumbs = useWorkspaceBreadcrumbs();
-
+  const { namespace } = useWorkspaceInfo();
   const [{ value: labels }] = useField<TriggerReleaseFormValues['labels']>('labels');
+  let applicationName = '';
+  const [releasePlans, loaded] = useReleasePlans(namespace);
+  const [selectedReleaseField] = useField('releasePlan');
```

Comment:
one nit pick: Can we change the name here to `selectedReleasePlanField` because Release is a different concept and can be confusing?

Reply:
Yeah that's fine, make sense

Reply:
Updated, please see: https://github.com/openshift/hac-dev/pull/990/files#diff-5793ae169199d7a4684f5fa9c14443c7b812b0881493d91c3c85e41fed823deeR32-R35